### ISSUE TITLE
FFmpeg nvidia encoder does not work through Juicify

### ISSUE THREAD

**Issue by haraldthi** [Juice-Labs/Juice-Labs#93]
There are a number of video tools that use Ffmpeg for input and output, and I'm having troubles with one of them: Topaz Video AI. I raised the issue on their forum (in this thread: https://community.topazlabs.com/t/running-video-ai-in-a-virtual-machine/53387/2 ) and I was asked to test if Ffmpeg could work as a standalone tool.

Indeed it doesn't fare so well. When the hardware encoder on the GPU is engaged, ffmpeg through juicify fails. The message being issued is typically:

[av1_nvenc @ 0000025ed351c340] Cannot load cuGetErrorString
exit status 0xc0000005
And logging the output of Juicify at level Trace doesn't add anything I can see of value.
Topaz Video AI spits out the log file attached here:
[logsForSupport.zip](https://github.com/Juice-Labs/Juice-Labs/files/12805223/logsForSupport.zip)

Is there a way that this can be fixed? It's hard to ask every video tool maker that uses Ffmpeg to step around this issue.

**Comment by Dema323** [comment]
Did you ever get this fixed, or found a workaround ?

**Comment by haraldthi** [comment]
No, it never was made a priority. The only workaround I have found is to reinstall my Windows partition as dual-boot.

**Comment by BlohoJo** [comment]
I am glad I found this thread because I was wanting to get Topaz Video AI to work on a Windows 10 VPS and found Juice Labs as a possible way to do it. Looks like it isn't possible, so this saved me on subscribing to it and also finding remote GPU's that I could rent.

Too bad it looks like no one is working on or updating this project anymore. :confused:
